CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ using MetricFu
 
 ## Unreleased
 
+* Prevent incompatible versions of metric_fu and flog to get installed together
```

Comment:
It's temporary but we are not sure for how long as the metric fu PR is still open.

Maybe we should add an observation to the changelog entry: if we release an version without an updated MetricFu version, this is an relevant change of this version; if it is released alongside an MetricFu update (and we restrict to this update version our own gemset) the changelog entry should get removed before the release.

It is not perfect but this may address both concerns. What do you think?